Misleading instructions after installation

After running the cookiecutter it says:
```
Now, enter the repository install the package:

pnpm i
make install

start coding, and push to your organization.
```

1. It's not very clear that I need to `cd name-of-my-project` before I run `pnpm i`.
2. I got an error when I tried to do that:
```
$ cd my-addon; pnpm i
Scope: all 2 workspace projects
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@plone/volto@workspace:*" is in the dependencies but no package named "@plone/volto" is present in the workspace

This error happened while installing a direct dependency of /Users/davisagli/Plone/my-addon

Packages found in the workspace: my-addon-dev, my-addon
Progress: resolved 1, reused 0, downloaded 0, added 0
```

--- a/volto_addon/hooks/post_gen_project.py
+++ b/volto_addon/hooks/post_gen_project.py
@@ -47,9 +47,9 @@ def main():
         f"""
         {_success('New add-on "{{ cookiecutter.addon_title }}" was generated')}
 
-        Now, enter the repository install the package:
+        Now, enter the generated directory and finish the install:
 
-        pnpm i
+        cd {{ cookiecutter.addon_title }}
         make install
 
         start coding, and push to your organization.

--- a/volto_addon/{{ cookiecutter.addon_name }}/.github/workflows/changelog.yml
+++ b/volto_addon/{{ cookiecutter.addon_name }}/.github/workflows/changelog.yml
@@ -49,9 +49,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pnpm i
           make install
-          pnpm i
 
       - name: Check for presence of a Change Log fragment (only pull requests)
         run: |

--- a/volto_addon/{{ cookiecutter.addon_name }}/README.md
+++ b/volto_addon/{{ cookiecutter.addon_name }}/README.md
@@ -87,7 +87,6 @@ Install package requirements
 
 ```shell
 make install
-pnpm i
 ```
 
 ### Start developing

